Dlls/LibraryDLL/.gitignore

```diff
@@ -0,0 +1,30 @@
+.vscode

```

Comment:
Please remove all these .gitignore files and add one in the root of the repository. You don't need to include every subproject name into it, this list will be enough:
```
Exe/
List/
Obj/
settings/
temp/
Debug/
Release/
include.tmp
*.dep
*.ewd
```

Reply:
Thank you, will fix that soon.